src/styles/core/typography/_font-styles.scss

```diff
@@ -194,5 +194,8 @@ $font-styles: (
   ),
   nav-menu-collapsible: (
     large: (nav_14_14, nav)
+  ),
+  on-this-page: (
```

Comment:
Can we just use `body-reduced-tight` instead of adding a new style since they seem to be the same?

Reply:
Sure can.